backend/src/main/java/de/neuefische/backend/controller/IndexCardController.java

```diff
@@ -0,0 +1,24 @@
+package de.neuefische.backend.controller;
+
+import de.neuefische.backend.model.IndexCard;
+import de.neuefische.backend.service.IndexCardService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("api/indexcard")
```

Comment:
```suggestion
@RequestMapping("/api/indexcard")
```
Bin mir nicht sicher ob es relevant ist, aber bisher haben wir, glaub ich immer ein / vor api geschrieben.